docs/source/backends/openwrt.rst

```diff
@@ -2285,6 +2285,231 @@ Will be rendered as follows::
+| key name        | type    | default      | description                                                 |
++=================+=========+==============+=============================================================+
+| ``network``     | string  | ``None``     | logical interface name (UCI specific),                      |
+|                 |         |              |                                                             |
+|                 |         |              | 2 to 15 alphanumeric characters, dashes and underscores     |
++-----------------+---------+--------------+-------------------------------------------------------------+
+| ``nohostroute`` | boolean | ``False``    | do not add routes to ensure the tunnel endpoints are routed |
+|                 |         |              | via non-tunnel device                                       |
++-----------------+---------+--------------+-------------------------------------------------------------+
+| ``fwmark``      | string  | ``None``     | firewall mark to apply to tunnel endpoint packets           |
++-----------------+---------+--------------+-------------------------------------------------------------+
+| ``ip6prefix``   | list    | ``[]``       | IPv6 prefixes to delegate to other interfaces               |
++-----------------+---------+--------------+-------------------------------------------------------------+
+| ``addresses``   | list    | ``[]``       | list of unique IPv4 or IPv6 addresses                       |
++-----------------+---------+--------------+-------------------------------------------------------------+
+
+The ``OpenWrt`` backend also adds ``wireguard_peers`` option for sepecifying a list of
+WireGuard Peers. It add the following properties to the ``wireguard_peers`` property of
+WireGuard schema.
+
++-----------------------+---------+-----------+-----------------------------------------------------------------+
+| key name              | type    | default   | description                                                     |
++=======================+=========+===========+=================================================================+
+| ``interface``         | string  | ``None``  | name of the wireguard interface,                                |
+|                       |         |           |                                                                 |
+|                       |         |           | 2 to 15 alphanumeric characters, dashes and underscores         |
++-----------------------+---------+-----------+-----------------------------------------------------------------+
+| ``route_allowed_ips`` | boolean | ``False`` | automatically create a route for each Allowed IPs for this peer |
++-----------------------+---------+-----------+-----------------------------------------------------------------+
+
+WireGuard example
+~~~~~~~~~~~~~~~~~
+
+The following *configuration dictionary*:
+
+.. code-block:: python
+
+    {
+        "interfaces": [{
```

Comment:
why the compressed `[{`?
Could you please keep consistency with the extendend formatting used in the other pages (have the curly brace break to the new line and indented further)?